### PR TITLE
new vendor form

### PR DESCRIPTION
**Pull request by localandopen**
getting my hands dirty with your server side based version

### CODE REVIEW

File: rust/src/templates/newvendor.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
+<html>
+	<head>
+        <title>Local-and-Open</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```
Comment (mawa) [inline rust/src/templates/newvendor.html]: Can we drop those and host a stylesheet locally?

File: rust/src/templates/newvendor.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        </div>
+
+    <div class="card-body">
+        <div class="card-subtitle mb-2 text-muted">
+           Aufnahme von ihrem Shop in die Datenbank
+        </div>
+
+        <div class=card-text>
+            Bitte f&uuml;llen Sie die folgenden Felder aus,
+            damit wir Ihren Shop entsprechend listen k&ouml;nnen.
```
Comment (mawa) [inline rust/src/templates/newvendor.html]: Haven't seen escaped Umlauts in a loooong time. :)

File: rust/src/templates/newvendor.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                    <div class="col">
+                    <input type="text" class="form-control" id="shopname" placeholder="Gesch&auml;ftsname">
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="branche">Branche</label>
+                    </div>
+                    <div class="col">
+                    <input type="text" class="form-control" id="branche" placeholder="Kinderspielzeug">
```
Comment (mawa) [inline rust/src/templates/newvendor.html]: We might want to offer a drop down / checkboxes for categories so we can enforce some structure on the catalog we're building.

File: rust/src/templates/newvendor.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="adress">Adresse</label>
+                    </div>
+                    <div class="col-4">
+                    <input type="text" class="form-control" id="address" placeholder="Adresse">
+                    </div>
+                    <div class="col">
+                    <input type="numeric" class="form-control" placeholder="Postleitzahl">
```
Comment (mawa) [inline rust/src/templates/newvendor.html]: Type should be `text`, a ZIP code isn't a number in the sense that you can do meaningful arithmetic on it.

File: rust/src/templates/newvendor.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                    </div>
+                    <div class="col">
+                    <input type="text" class="form-control" placeholder="Stadt">
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-35">
+                        <div class="form-check">
+                            <input type="checkbox" class="form-check-input" id="exampleCheck1">
+                            <label class="form-check-label" for="exampleCheck1">Hiermit stimmen Sie zu, dass Sie berechtigt sind, die hier angegebenen Daten zur Verfuegung zu stellen</label>
```
Comment (mawa) [inline rust/src/templates/newvendor.html]: Umlaut.

File: rust/src/templates/newvendor.html (hunk 1 above)
Comment (localandopen) [reply rust/src/templates/newvendor.html]: yep

File: rust/src/templates/newvendor.html (hunk 2 above)
Comment (localandopen) [reply rust/src/templates/newvendor.html]: and i don't know better ;)

File: rust/src/templates/newvendor.html (hunk 3 above)
Comment (localandopen) [reply rust/src/templates/newvendor.html]: yes, you are right, didn't have time for it

File: rust/src/templates/newvendor.html (hunk 4 above)
Comment (localandopen) [reply rust/src/templates/newvendor.html]: just wanted to have some simple input verification

**Pull request merged**